app/controllers/api/v1/search_skills_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::SearchSkillsController < ApplicationController
+  before_action :searched_users, only: [:index]
+  before_action :check_for_empty_query
```

Comment:
I like this; being able to check if there is empty query first and giving an error message first before any action is even done yet. Nice!